atom/browser/atom_browser_client.cc

```diff
@@ -82,7 +86,7 @@ namespace {
 bool g_suppress_renderer_process_restart = false;
 
 // Custom schemes to be registered to handle service worker.
-std::string g_custom_service_worker_schemes = "";
+base::NoDestructor<std::string> g_custom_service_worker_schemes;
```

Comment:
Asking for my information:
Why is this change needed? And when should it be used?

Reply:
Using the base::NoDestructor<> wrapper to make it explicit that these variables are never destroyed, fixing `global string variables are not permitted` warnings.

The style guide's take on globals with nontrivial destructors: https://google.github.io/styleguide/cppguide.html#Static_and_Global_Variables